addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/station/NAMainHandler.java

```diff
@@ -44,7 +44,7 @@ protected NAMain updateReadings() {
             userAdministrative = stationDataBody.getUser().getAdministrative();
 
             result = stationDataBody.getDevices().stream().filter(device -> device.getId().equalsIgnoreCase(getId()))
-                    .findFirst().get();
+                    .findFirst().orElse(null);
```

Comment:
This was one of the real bugs, get throws an exception when you call it on an empty Optional

Reply:
For my personal interest: You asked me in one of my last PRs to handle the `Optional` instead of using the `orElse()`
method (see  https://github.com/openhab/openhab2-addons/pull/2820#discussion_r159368313). Why not in this case?

Reply:
Fair question, in this case it was the quick way out, a get was already done and combined with a null check, but get cannot return null but instead will return an exception. Also there was an api here which expected a value or null.

So the summary is that I was to lazy to change the surrounding code 😏

Reply:
I somehow expected that answer. ;-) I'm fine with that.